public/components/common/welcome/components/menu-agent.js

```diff
@@ -26,88 +27,165 @@ class WzMenuAgent extends Component {
       hoverAddFilter: '',
     };
 
-    this.menuAgent = window.localStorage.getItem('menuAgent') ? JSON.parse(window.localStorage.getItem('menuAgent')) : {};
+    this.menuAgent = window.localStorage.getItem('menuAgent')
+      ? JSON.parse(window.localStorage.getItem('menuAgent'))
+      : {};
 
     this.agentSections = {
```

Comment:
These sections could we get them from another side? to reduce the "lines" in the component?

Reply:
done!